lib/supplejack/search.rb

```diff
@@ -299,14 +299,17 @@ def execute_request
         else
           @response = get(request_path, @api_params)
         end
+      rescue RestClient::ServiceUnavailable => e
```

Comment:
you can have multiple errors in a single rescue so just move it up and have a `,` between them.(from memory)

Reply:
👍